### PR TITLE
Drop support for EOL Python 2.7

### PR DESCRIPTION
**Pull request by hugovk**
For https://github.com/marshmallow-code/webargs/issues/440.

### CODE REVIEW

**Comment by lafrech** [comment]
Thank you so much for your work on this, @hugovk.

**Comment by hugovk** [comment]
You're welcome! Python 3.5 failed, I guess it needs some backports the non-stdlib `mock` package, but will check back in a bit.

**Comment by hugovk** [comment]
Updated and now ✅!

**Comment by sloria** [comment]
@hugovk  Can you resolve the conflicts?

Once that's done and 6.0.0b1 is released, I think we're good to merge and release this cc @lafrech

**Comment by hugovk** [comment]
Updated and now ✅!

File: src/webargs/core.py [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-
 import functools
 import inspect
 import logging
 import warnings
 from copy import copy
+from collections.abc import Mapping
 
 try:
     import simplejson as json
```
Comment (lafrech) [inline src/webargs/core.py]: I suspect this could be removed.

**Review by lafrech — APPROVED** [review]
Looks good to me.

I suspect `simplejson` was used in PY2 to compensate for shortcomings in core `json`.

We should also test against 3.8. Let's do that in another PR.

**Comment by sloria** [comment]
Yeah, the simplejson import can be removed, as it was only for Python 2 compat

https://github.com/marshmallow-code/webargs/blob/c90e814f4dc1624586cd04180f553a6c4c77f873/setup.py#L7-L8

**Comment by hugovk** [comment]
Does that mean we should use universal wheels?

```diff
diff --git a/setup.cfg b/setup.cfg
 [bdist_wheel]
-# We build separate wheels for
-# Python 2 and 3 because of the conditional
-# dependency on simplejson
-universal = 0
+universal = 1
```

File: src/webargs/core.py (hunk 1 above)
Comment (hugovk) [reply src/webargs/core.py]: Removed in b734c5508f63cea93f3f6cee34807f1d400d2fe9!

**Comment by sloria** [comment]
Yes, good catch. We can build universal wheels now.

**Comment by hugovk** [comment]
Updated!

**Comment by lafrech** [comment]
Great. Let's move on with this.

**Comment by lafrech** [comment]
Released as 6.0.0b2 !

Thanks again @hugovk.

**Comment by hugovk** [comment]
You're welcome!

**Pull request merged**